synapse/models/infotech.py

```diff
@@ -540,6 +627,180 @@ def getModelDefs(self):
+                        'doc': 'The URL that documents the ATT&CK tactic.',
+                    }),
+                    ('tag', ('syn:tag', {}), {
+                        'doc': 'The synapse tag used to annotate nodes included in this ATT&CK tactic.',
+                        'ex': 'cno.mitre.ta0100',
+                    }),
+                    ('references', ('array', {'type': 'inet:url'}), {
+                        'doc': 'An array of URLs that document the ATT&CK tactic.',
+                    }),
+                )),
+                ('it:mitre:attack:technique', {}, (
+                    ('name', ('str', {'strip': True}), {
+                        'doc': 'The primary name for the ATT&CK technique.',
+                    }),
+                    ('desc', ('str', {'strip': True}), {
+                        'doc': 'A description of the ATT&CK technique.',
+                        'disp': {'hint': 'text'},
+                    }),
+                    ('url', ('inet:url', {}), {
+                        'doc': 'The URL that documents the ATT&CK technique.',
+                    }),
+                    ('tag', ('syn:tag', {}), {
+                        'doc': 'The synapse tag used to annotate nodes included in this ATT&CK technique.',
+                        'ex': 'cno.mitre.t0100',
+                    }),
+                    ('references', ('array', {'type': 'inet:url'}), {
+                        'doc': 'An array of URLs that document the ATT&CK technique.',
+                    }),
+                    ('parent', ('it:mitre:attack:technique', {}), {
+                        'doc': 'The parent ATT&CK technique on this sub-technique.',
+                    }),
+                )),
+                ('it:mitre:attack:software', {}, (
+                    ('software', ('it:prod:soft', {}), {
+                        'doc': 'Used to map an ATT&CK software to a synapse it:prod:soft.',
+                    }),
+                    ('name', ('str', {'strip': True}), {
+                        'doc': 'The primary name for the ATT&CK software.',
+                    }),
+                    ('desc', ('str', {'strip': True}), {
+                        'doc': 'A description of the ATT&CK software.',
+                        'disp': {'hint': 'text'},
+                    }),
+                    ('url', ('inet:url', {}), {
+                        'doc': 'The URL that documents the ATT&CK software.',
+                    }),
+                    ('tag', ('syn:tag', {}), {
+                        'doc': 'The synapse tag used to annotate nodes included in this ATT&CK software.',
+                        'ex': 'cno.mitre.s0100',
+                    }),
+                    ('references', ('array', {'type': 'inet:url'}), {
+                        'doc': 'An array of URLs that document the ATT&CK software.',
+                    }),
+                    ('techniques', ('array', {'type': 'it:mitre:attack:technique'}), {
```

Comment:
Also part of discussion re: pros / cons of cross-ref'ing groups / software / techniques.